pkgs/tools/admin/mcumgr/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, mcumgr }:
+
+buildGoModule rec {
+  pname = "mcumgr";
+  version = "unstable-2022-10-04";
```

Comment:
```suggestion
  version = "0-unstable-2022-10-04";
```

Reply:
is this a new style change? did not see it like that before